modules/apps/frontend-js/frontend-js-remote/tests/RemoteAppliactions.testcase

```diff
@@ -0,0 +1,82 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-js-components-sample-web";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Remote Applications";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		PortalInstances.tearDownCP();
+	}
+
+	@description = "LPS-163527. Assert iFrame is not add and warnings appear."
+	@priority = "4"
+	test AssertErrorMessegeAppearsWhenIFrameFieldsEmpty {
+		task ("Given Add IFrame Remote Apps admin page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Custom Apps",
+				panel = "Applications",
+				portlet = "Remote Apps");
+
+			RemoteApplications.AddiFrame();
+		}
+
+		task ("When required fields Name and URL are blank and When attempt to save remote app") {
```

Comment:
`When required fields Name and URL are blank` can be it's own macro to clear the fields to make sure they're blank.